gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
Just a question: here you use the 8.4 gradle version and in the version catalog 8.2.0 is used. Does this conflict and was this a choice?

Reply:
This does not seem to conflict, also not a choice, it was autogenerated like this. The highest I could bump it to in the version catalog is 8.2.2 which I just did, not sure why 8.4 can't be found